addons/web/static/tests/views/kanban_view_tests.js

```diff
@@ -10000,6 +10000,34 @@ QUnit.module("Views", (hooks) => {
         }
     );
 
+    QUnit.test("keyboard navigation on kanban when the kanban has a oe_kanban_global_click class", async (assert) => {
+
+        assert.expect(1);
+        await makeView({
+            type: "kanban",
+            resModel: "partner",
+            serverData,
+            arch:
+                "<kanban>" +
+                '<templates><t t-name="kanban-box">' +
+                '<div class="oe_kanban_global_click">' +
+                '<field name="name"/>' +
+                "</div>" +
+                "</t></templates>" +
+                "</kanban>",
```

Comment:
We should add a button to show, it is not this one it will be triggered and clicked when we hit `enter` during the keyboard navigation.

Also, you can reformat like this:
```suggestion
                `<kanban>
                    <templates>
                        <t t-name="kanban-box">
                            <div class="oe_kanban_global_click">
                                <field name="name"/>
                            </div>
                            <a name="action_test" type="object" />
                        </t>
                    </templates>
                 </kanban>`,
```